src/pages/DashBoard/DashBoard.tsx

```diff
@@ -1,11 +1,39 @@
-import { useState } from 'react';
+import { useState, useEffect, FC } from 'react';
+import { TOTAL_DAYS } from 'constants/index';
 import DashboardTable from 'components/DashboardTable';
 import Container from 'components/Container';
 import DashboardNavigation from 'components/DashboardNavigation';
+import Calendar from 'components/Calendar';
+import getCalendarDays from 'helpers/getCalendarDays';
 
-const DashBoard = () => {
+interface CalendarDay {
+  day: number;
+  isWeekend: boolean;
+  isToday: boolean;
+}
+
+interface Vacation {
+  id: string;
+  vacationType: string;
+  startDate: string;
+  endDate: string;
+  note: string;
+}
+
+interface Vacations {
```

Comment:
duplication

define those interfaces in a place where you will store them and export from there.

Reply:
done